ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.20-nightly-x86.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cerberus:
+    name: cerberus
+    namespace: chaos
+    tag: cerberus-prow
+  ocp-qe-perfscale-ci:
+    name: ocp-qe-perfscale-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.20"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: bgp
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perscale-cpt
+    env:
+      HV_VM_CPU_COUNT: "16"
+      HV_VM_DISK_SIZE: "120"
+      HV_VM_DISK1_CONFIG: 'default: 1'
+      HV_VM_DISK2_CONFIG: 'sdb: 1'
+      HV_VM_DISK3_CONFIG: 'nvme0n1: 2'
+      HV_VM_MEMORY_SIZE: "32"
+      NUM_NODES: "9"
+      NUM_WORKER_NODES: "24"
+      PUBLIC_VLAN: "false"
+      STARTING_NODE: "1"
+      TYPE: vmno
+      VMNO_HV_VM_JETLAG_DEFAULTS: "false"
+    test:
+    - ref: openshift-qe-installer-bm-ping
+    - ref: openshift-qe-installer-bm-poweroff
+    - ref: openshift-qe-installer-bm-foreman
+    - ref: openshift-qe-installer-bm-prereqs
+    - ref: openshift-qe-installer-bm-deploy
+    - ref: openshift-qe-cluster-health
+    - chain: openshift-qe-udn-bgp
```

Comment:
Do we need a chain with several refs? 
If we only have one ref please use ref instead of chain and delete the chain